addons/oci-finops/finops-setup/finops_iam.auto.tfvars.json

```diff
@@ -0,0 +1,82 @@
+{
+  "compartments_configuration": {
+    "enable_delete": "true",
+    "compartments": {
+      "CMP-LANDINGZONE-P-KEY": {
+        "name": "cmp-landingzone-p",
+        "description": "Enclosing Production Landing Zone Compartment",
+        "children": {
+          "CMP-LZP-NETWORK-KEY": {
+            "name": "cmp-lzp-network",
+            "description": "Shared Network Compartment"
+          },
+          "CMP-LZP-SECURITY-KEY": {
+            "name": "cmp-lzp-security",
+            "description": "Shared Security Compartment"
+          },
+          "CMP-LZP-PLATFORM-KEY": {
+            "name": "cmp-lzp-platform",
+            "description": "Shared Platform Compartment",
+            "children": {
+              "CMP-LZP-PLATFORM-FINOPS-KEY": {
+                "name": "cmp-lzp-platform-finops",
+                "description": "Finops Platform Environment"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "groups_configuration": {
+    "groups": {
+      "GRP-LZP-PLATFORM-FINOPS-ADMINS": {
+        "name": "grp-lzp-platform-finops-admins",
+        "description": "Production Landing Zone Environment FinOps platform administration group, including common OE network elements."
+      }
+    }
+  },
+  "dynamic_groups_configuration": {
+    "default_defined_tags": null,
+    "default_freeform_tags": null,
+    "dynamic_groups": {
+      "DG-LZP-PLATFORM-FINOPS-ADB": {
+        "name": "dg-lzp-platform-finops-adb",
+        "description": "Dynamic group for Autonomous Databases in the lzp-platform-finops compartment.",
+        "matching_rule": "ALL {resource.type = 'autonomousdatabase', resource.compartment.id = 'CMP-LZP-PLATFORM-FINOPS-KEY'}"
```

Comment:
I don't think this will get replaced automatically, check in the IAM module. If it doesn't overload the matching rules then replace with a placeholder that needs to be updated manually for applying